src/server.prod.js

```diff
@@ -58,8 +68,15 @@ class ProdServer {
     const clientPublicPath = this.config.client.output.publicPath;
     const serverOutputPath = this.config.server.output.path;
     const clientOutputPath = this.config.client.output.path;
-    app.use(serverPublicPath, express.static(serverOutputPath));
-    app.use(clientPublicPath, express.static(clientOutputPath));
+
+    if(this.config.compressAssets) {
```

Comment:
```suggestion
    if (this.config.compressAssets) {
```

Sorry for nitpicking